src/airodump-ng/airodump-ng.c

```diff
@@ -808,6 +833,8 @@ static const char usage[] =
 	"      --ht40-               : Set channel to HT40- (802.11n)\n"
 	"      --ht40+               : Set channel to HT40+ (802.11n)\n"
 	"      --channel <channels>  : Capture on specific channels\n"
+	"      --ignore-other-chans  : Filter out other channels,\n"
+	"                              can be used with --channel\n"
```

Comment:
Could you reword this: "Requires --channel (or -c)" (and remove the comma above.

Reply:
updated, please check again